doc/changelog/03-notations/14819-rm_deprecated_813.rst

```diff
@@ -0,0 +1,32 @@
+- **Removed:**
+  Removed the ``Numeral Notation`` command that was renamed to :cmd:`Number Notation` in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  Removed primitive float notations ``<``, ``<=`` and ``==`` that were replaced by ``<?``, ``<=?`` and ``=?`` in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  Removed the file Numeral.v that was replaced by Number.v in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  Removed primitive integer notations ``\%``, ``<``, ``<=`` and ``==`` that were replaced by ``mod``, ``<?``, ``<=?`` and ``=?`` in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  Remove some ``*_invol`` functions that were renamed ``*_involutive`` for consistency with the remaining of the stdlib in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  Remove boolean attributes ``monomorphic``, ``noncumulative`` and ``notemplate```that were replaced by ``polymorphic=no``, ``cumulative=no`` and ``template=no`` in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  Remove command ``Grab Existential Variables`` replaced by :cmd:`Unshelve` (mostly equivalent, up to the reverse order of the resulting subgoals) in 8.13.
```

Comment:
Maybe say "deprecated in 8.13" (because like this it reads as if `Unshelve` had been introduced in 8.13).